challenge.md

```diff
@@ -31,19 +31,19 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
-<!-- Answer Starts Here -->
+Absolute. It is exactly where you will find it.<!-- Answer Starts Here -->
```

Comment:
You're correct that this is an absolute path, but your explanation of why isn't very meaningful. 

The reason that this is an absolute path is because it's defining the position of `Desktop` with respect to the `root`, a fixed position within the filesystem. This path will always show someone how to find the `Desktop` directory, irrespective of where they are in the filesystem.